ts/src/kucoin.ts

```diff
@@ -1760,6 +1760,18 @@ export default class kucoin extends Exchange {
         return orderbook;
     }
 
+    handleTriggerPrices (params) {
+        const triggerPrice = this.safeValue2 (params, 'triggerPrice', 'stopPrice');
+        const stopLossPrice = this.safeValue (params, 'stopLossPrice');
+        const takeProfitPrice = this.safeValue (params, 'takeProfitPrice');
+        const isStopLoss = stopLossPrice !== undefined;
+        const isTakeProfit = takeProfitPrice !== undefined;
+        if ((isStopLoss && isTakeProfit) || (triggerPrice && stopLossPrice) || (triggerPrice && isTakeProfit)) {
+            throw new ExchangeError (this.id + ' createOrder() - you should use either triggerPrice or stopLossPrice or takeProfitPrice');
+        }
```

Comment:
@ttodua actually I'm not a big fan of this return type, it is very easy to mess up the order upon calling this method

Reply:
but the method itself is useful, so we can keep it for now I think

Reply:
I've intentionally started dissecting that logic/method from 'createOrder' implementations itself, because that common pattern/parser can be moved into base later, with like `this.handleStopLossTakeProfit` or whatever, and can be reused in all derived classes. but atm, I'm still observing them and yet adding inside same derived classes. later, I'll have observed more commonalities.